xml/System.IO.Packaging/Package.xml

```diff
@@ -1241,7 +1241,7 @@
 ## Remarks  
  `id` must be a valid XML identifier.  The `id` type is xsd:ID and must follow the naming conventions prescribed in the XML Schema Part 2: Datatypes specification (see [https://www.w3.org/TR/xmlschema-2/#ID](https://www.w3.org/TR/xmlschema-2/#ID)).  
```

Comment:
There's a duplication of link text and link here that can be removed with angle-brackets.